setup.cfg

```diff
@@ -44,6 +44,7 @@ install_requires =
     squidpy
     napari[all] @ git+https://github.com/napari/napari.git#egg=napari[all]
     loguru
+    napari-matplotlib
```

Comment:
is there a way to have napari-matplotlib as a dependency, but not showing it up in the plugin bar? 
<img width="217" alt="image" src="https://user-images.githubusercontent.com/25887487/191514292-f5a8917e-ec89-48b4-93b3-cc8f0adb7ff1.png">

it's no problem, but I don't think the user should interact with it (as it interacts via our interface)/

Reply:
Kevin (@kevinyamauchi) mentioned that this isn't fixable right now but we can create an issue in the napari github repo. (I'll create one there)